pages/journal/index.tsx

```diff
@@ -297,7 +297,7 @@ export async function getStaticProps(context: GetStaticPropsContext) {
         props: {
             journalEntries: data,
         },
-        revalidate: globals.revalidate.journal,
+        revalidate: globals.revalidate.home,
```

Comment:
should this also be changed back to `.journal`

Reply:
Good eye 👀